docs/hub/security-pickle.md

```diff
@@ -1,20 +1,5 @@
 # Pickle Scanning
 
-- [TL;DR](#tldr)
-- [What is a pickle?](#what-is-a-pickle)
-- [Why is it dangerous?](#why-is-it-dangerous)
-- [Mitigation Strategies](#mitigation-strategies)
-  * [Load files from users and organizations you trust](#load-files-from-users-and-organizations-you-trust)
-  * [Load model weights from TF or Flax](#load-model-weights-from-tf-or-flax)
-  * [Use your own serialization format](#use-your-own-serialization-format)
-  * [Improve `torch.load/save`](#improve-torchloadsave)
-  * [Hub’s Security Scanner](#hubs-security-scanner)
-    + [What we have now](#what-we-have-now)
-    + [Potential solutions](#potential-solutions)
-- [Further Reading](#further-reading)
-
-## TL;DR
```

Comment:
Not an ideal title for the search

<img width="1702" alt="image" src="https://user-images.githubusercontent.com/3841370/192745424-38be7f9a-0d63-4a82-ae95-b228a3d8afd0.png">